pkg/config/piped.go

```diff
@@ -345,6 +351,13 @@ func (g PipedGit) LoadSSHKey() ([]byte, error) {
 	return nil, errors.New("either sshKeyFile or sshKeyData must be set")
 }
 
+func (g *PipedGit) Validate() error {
+	if g.ShouldConfigureSSHConfig() && g.PersonalAccessToken.ShouldConfigurePATConfig() {
```

Comment:
`ShouldConfigureSSHConfig()` is not exactly for validation purpose(called [here](https://github.com/pipe-cd/pipecd/blob/315748e42b1a5fc29d39a19bccebc5c27f25d254/pkg/app/piped/cmd/piped/piped.go#L159)). Can you validate ssh and pat config without using `ShouldConfigureXXX()`?